packages/language-server/src/server.ts

```diff
@@ -25,15 +22,35 @@ import {
   hasSettingsForDocument,
   getSettingsForDocument,
 } from "@ui5-language-assistant/settings";
+import { ServerInitializationOptions } from "../api";
+import { getSemanticModel } from "./ui5-model";
+import { getCompletionItems } from "./completion-items";
+import { getXMLViewDiagnostics } from "./xml-view-diagnostics";
+import { getHoverResponse } from "./hover";
+import {
+  getFlexEnabledFlagForXMLFile,
+  isManifestDoc,
+  initializeManifestDocuments,
+  updateManifestData,
+} from "./manifest-handling";
 
 const connection = createConnection(ProposedFeatures.all);
 const documents = new TextDocuments(TextDocument);
 let getSemanticModelPromise: Promise<UI5SemanticModel> | undefined = undefined;
+let getInitialManifestStatePromise: Promise<void[]> | undefined = undefined;
 let initializationOptions: ServerInitializationOptions | undefined;
 let hasConfigurationCapability = false;
 
 connection.onInitialize((params: InitializeParams) => {
   const capabilities = params.capabilities;
+  const workspaceFolderUri = params.rootUri;
+  if (workspaceFolderUri) {
+    const workspaceFolderPath = URI.parse(workspaceFolderUri).fsPath;
```

Comment:
subjective:

Do we want to emphasize that this is an absolute path because of future
multi root scenarios that would require absolute paths?

- `abs` suffix/prefix/?